dbt/models/iasworld/schema/iasworld.asmt_all.yml

```diff
@@ -75,16 +486,25 @@ sources:
             description: '{{ doc("column_whocalc") }}'
 
         tests:
+          # Note that this test is NOT actually the unique primary key of
+          # this table, since there doesn't seem to BE a unique combination
+          # of identifying columns
```

Comment:
**[Question, non-blocking]** Hm, does that mean we should just remove this test then?

Reply:
I think it's still useful, since an error firing would indicate an increasing number of duplicate values.